tests/Oryx.BuildImage.Tests/VersionInformationTest.cs

```diff
@@ -226,12 +226,10 @@ public void Node_UsesLTSVersion_ByDefault_WhenNoExplicitVersionIsProvided(string
         [InlineData("8", "v" + NodeVersions.Node8Version)]
         [InlineData("10", "v" + NodeVersions.Node10Version)]
         [InlineData("12", "v" + NodeVersions.Node12Version)]
-        [InlineData("14", "v" + NodeVersions.Node14Version)]
```

Comment:
I am suspecting there are lot of other places where node 14 is testing

Reply:
yeah .. thats why I triggered a nightly build  to find out the failed tests .. will update this PR accordingly